x-pack/plugins/security_solution/public/management/components/console/components/command_input/command_input.tsx

```diff
@@ -237,11 +253,16 @@ export const CommandInput = memo<CommandInputProps>(({ prompt = '', focusRef, ..
 
   return (
     <InputAreaPopover width={popoverWidth}>
-      <CommandInputContainer {...commonProps} onClick={handleTypingAreaClick} ref={containerRef}>
+      <CommandInputContainer
+        {...commonProps}
+        className={focusClassName}
+        onClick={handleTypingAreaClick}
+        ref={containerRef}
+      >
         <EuiFlexGroup
           wrap={true}
           responsive={false}
-          alignItems="flexStart"
+          alignItems="center"
```

Comment:
Here and below: why did you need to set the items alignment to `center`?
I really dont' think that is the desired alignment. Everything should be aligned to the left (start) of the flex group.

Reply:
ahhh... yes, you are right. Thanks pointing that out. I just looked it up and it "...it controls the alignment of items on the Cross Axis.". I always get it confused with `align-context`.

Reply:
yea, can confirm in a before/after that the alignment of text in the text looks much better with this PR.

before:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/56395104/176774700-5904d7dc-16bd-44b5-84eb-f71354ee70eb.png">

after:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/56395104/176774585-8aef3517-791e-4b89-9f6e-4855c01f00f6.png">